docs/layouts/shortcodes/code.html

```diff
@@ -0,0 +1,42 @@
+{{/*
```

Comment:
This looks pretty similar to https://github.com/osohq/oso/blob/bf24dd53b9365ed70d359e56f25c5c567bae1b5d/docs/themes/oso-tailwind/layouts/shortcodes/code.html

Maybe we can combine?

Reply:
Oh yikes I didn't know that existed -- what's the difference between the two shortcode directories?

Reply:
I honestly don't remember lol — maybe @samscott89 does